docs/git-node.md

```diff
@@ -330,7 +335,7 @@ ncu-config set waitTimeMultiApproval 48
 
 ## `git node v8`
 
-Update or patch the V8 engine.  
+Update or patch the V8 engine.
```

Comment:
revert it? I think I got the same behaviour when using npm run lint-fix.

Reply:
Refs: https://www.markdownguide.org/basic-syntax/#line-breaks

Reply:
```suggestion
Update or patch the V8 engine.  
```